types/src/state_store/state_value.rs

```diff
@@ -9,14 +9,15 @@ use aptos_crypto::{
     HashValue,
 };
 use aptos_crypto_derive::{BCSCryptoHash, CryptoHasher};
+use once_cell::sync::OnceCell;
```

Comment:
why sync?

Reply:
we need thread-safe for most of our things, I don't think we use unsync anywhere